ftl/core/preferences.ftl

```diff
@@ -46,3 +46,6 @@ preferences-daily-backups = Daily backups to keep:
 preferences-weekly-backups = Weekly backups to keep:
 preferences-monthly-backups = Monthly backups to keep:
 preferences-minutes-between-backups = Minutes between automatic backups:
+preferences-appearance = <b>Appearance</b>
```

Comment:
The only other bold text in that screen is the Synchronization label. Could we do without the label? You could argue other things like "show play buttons" also alter appearance too.

Semi-related: 
https://translating.ankiweb.net/anki/developers.html#avoid-html-where-possible

Like the fields screen, I'd prefer we add new options below instead of to the right